### PR TITLE
HW18 completed

### PR DESCRIPTION
**Pull request by crpro108**

### CODE REVIEW

**Comment by Berichicko** [comment]
Please finish the homework!You got this problem, please correct it.

image

image

HINT

How to analyze the results of tests:

Test status: 
Test status_en

	
Test error (red). This status is assigned to tests that caused an exception from the tested source code.
Test error" means there's a problem with the code being tested. This is not okay for a tester. We need to fix it to make sure the software works correctly
Test error_en

Test failed(yellow). If at least one test receives this status, then all its parents are marked as failed.
Test failed_en

SOLUTION



or

File: src/test/java/homework18.java [hunk 1]
```diff
@@ -0,0 +1,36 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.chrome.ChromeOptions;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.time.Duration;
+public class homework18 extends BaseTest{
```
Comment (Berichicko) [inline src/test/java/homework18.java:10]: **According to the Java style guide:** 

1. Class names should be nouns, in mixed cases with the first letter of each internal word capitalized. Interfaces names should also be capitalized just like class names.
2. Use whole words and must avoid acronyms and abbreviations.

For example HomeWork18

File: src/test/java/homework17.java [hunk 2]
```diff
@@ -12,28 +12,28 @@ public class homework17 extends BaseTest{
     @Test
```
Comment (Berichicko) [inline src/test/java/homework17.java:12]: **According to the Java style guide:** 

1. Class names should be nouns, in mixed cases with the first letter of each internal word capitalized. Interfaces names should also be capitalized just like class names.
2. Use whole words and must avoid acronyms and abbreviations.

For example HomeWork17

**Comment by crpro108** [comment]
Thank you Alex

File: src/test/java/homework18.java (hunk 1 above)
Comment (crpro108) [reply src/test/java/homework18.java:10]: I will adjust this in the next assignments, thank you

**Comment by Berichicko** [comment]
Hello,

I have checked your homework 18, and I wanted to let you know that you're doing great! Keep up the excellent work!
Best wishes from the HW team (Alex, [aliaksandr@testpro.io](mailto:aliaksandr@testpro.io))!

804deb64dc406a20a75da05d91a1a69e

image

**Pull request merged**